### ISSUE TITLE
nvda and espeak-ng problem announcing some punctuation when Punctuation/symbol level: is set to none

### ISSUE THREAD

**Issue by zahra21** [nvaccess/nvda#6514]
hello every one.
when i tested nvda 2016.3 i faced this problem with espeak and punctuations.
i set my nvda in voice settings to read farsi documents.
when punctuations are between two words and there is not space between punctuations and second word, unfortunately nvda announces the punctuation!
steps to reproduce:
1/ use nvda preferences and voice settings.
2/ in the voice settings, set nvda in to persian.
3/ also in the voice settings, set ponctuation to none.
4/ press enter to save the changes.
5/ copy this farsi words which kama is between them.
من،دوستم

expected behaviour: nvda does not announce any ponctuation when i set nvda Punctuation/symbol level: none
actual result: nvda reads ponctuation if there is not space between ponctuation and the next word.

**Comment by jcsteh** [comment]
Does this happen in NVDA 2016.2.1?

This is an issue in eSpeak NG and needs to be fixed there. The question is whether it is a recently introduced bug and perhaps whether it's already been fixed.

**Comment by zahra21** [comment]
> hi.
> yes. i should mention that the issues for punctuation in farsi is only happen in espeak ng and not its previous versions of espeak.
> in nvda 2016.2.1 the situation was worse and nvda speaks punctuation every time specially right paren.
> but ist is solved fortunatelysee bug 6037
> https://github.com/nvaccess/nvda/issues/6037
> but i want to nvda does not announce any punctuation in any time. 
> because i set punctuation symbel level to none and i expect to dont hear any punctuation.
> thanks extremely for your reply and helping me.
> could you please fix it in nvda 2016.4 to does not announce punctuations when settings punctuation symbel level to none in voice settings?

**Comment by jcsteh** [comment]
Unfortunately, we can't fix this ourselves. This needs to be fixed in eSpeak NG by the Farsi maintainer.

**Comment by bhavyashah** [comment]
@zahra21  Is this pronunciation issue still existent? If yes, please report it on the tracker part of ESpeak-NG's Github project located at https://github.com/espeak-ng/espeak-ng.

**Comment by zahra21** [comment]
hello.
in nvda 2017.1 and 2017.2 which use espeak-ng 1.49.1.dev the issues is still persists.
but i could not espeak-ng 1.49.1 stable sapi5 version to test.
also, i downloaded one version of espeak-ng which is not sapi5 version, but i could not use it in my system windowsXP 32bit.
although i downloaded x86 version, when i tried to open it, said not valid for win32!
could you please test with nvda master and give me the result?
i could not download nvda master and recieve error every time.
just do the following.
1- in nvda preferences voice settings, select Persian+English-US or Persian+English-UK and confirm the change.
then see this word.
does nvda announce ويرگول؟
heres the word.
ميگويد،و
please also dont forget to set Punctuation/symbol level: none
in voice setting to test this case.

**Comment by bhavyashah** [comment]
@rhdunn Could you please take a look at this ESpeak-NG issue reported here by @zahra21? Also, it would be appreciated if you could either assist Zahra in filing a ticket on the ESpeak-NG Github tracker or alternatively notify the Farsi maintainer of ESpeak-NG about this problem. Also, CC @jcsteh in case you may want to address https://github.com/nvaccess/nvda/issues/6514#issuecomment-319882748.

**Comment by Adriani90** [comment]
@zahra21 do you still have this issue with NVDA 2019.1 RC1?

**Comment by zahra21** [comment]
i dont know, because you know that i use windows xp and 2019.1 cant be
installed on my windows to test!
by the way, i need sapi5 version of espeak ng 1.49.1, where can i find it?

On 3/18/19, Adriani90 <notifications@github.com> wrote:
> @zahra21 do you still have this issue with NVDA 2019.1 RC1?
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/6514#issuecomment-474093162


-- 
By God,
were I given all the seven heavens
with all they contain
in order that
I may disobey God
by depriving an ant
from the husk of a grain of barley,
I would not do it.
imam ali

**Comment by Adriani90** [comment]
It seems the last eSpeak version compiled for sapi5 on Windows is 1.48.04:
http://espeak.sourceforge.net/download.html
eSpeak 1.49.1 and 1.49.2 do not support Sapi5 voices yet.

I am closing this issue because we lack of contextual testing from the initial author. If we have a confirmation that the issue still occurs, we can reopen it. Thanks.
